client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-select-checkbox/index.tsx

```diff
@@ -17,15 +17,15 @@ export default function SiteSelectCheckbox( { item, siteError, isLargeScreen }:
 	);
 
 	function handleCheckboxClick() {
-		item.onSelect( item );
+		item.onSelect?.();
```

Comment:
`item` is not required since we are not using it anywhere. You can verify it here - https://github.com/Automattic/wp-calypso/blob/update/refactor-ts-types-in-pro-dashboard/client/jetpack-cloud/sections/agency-dashboard/sites-overview/index.tsx#L83